components/brave_wallet/browser/ethereum_keyring.cc

```diff
@@ -34,8 +33,9 @@ bool EthereumKeyring::RecoverAddress(const std::vector<uint8_t>& message,
                                      std::string* address) {
   CHECK(address);
   // A compact ECDSA signature (recovery id byte + 64 bytes).
-  if (signature.size() != 65)
+  if (signature.size() != kCompactSignatureSize + 1) {
```

Comment:
@kdenhartog is having a compact signature safe here?

Reply:
Thanks for calling this out @thypon

TL:DR - yeah compact signatures are considered safe and in this case it looks like it's just reusing a constant.

No concerns here from me